backend/src/api/conversions.clj

```diff
@@ -0,0 +1,23 @@
+(ns api.conversions)
+
+(defn player-num
+  "Translates a Player id into an internal player representation"
+  [game-state id] (.indexOf (:player-ids game-state) id))
+
+(defn translate-player
+  "Translates an internal player to a human-readable representation"
+  [game-state internal me]
+  (cond
+    (nil? internal) nil
+    (>= internal 2) :tie
+    (= me (get-in game-state [:player-ids internal])) :me
+    :else :opponent))
+
+(defn ^:private uuid [] (str (java.util.UUID/randomUUID)))
+
+(defn create-player
```

Comment:
`generate-player-uuid`? Also, this doesn't fit as a "conversion", maybe on its own file?